draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1110,25 +1129,17 @@ whose output distribution is not uniformly random, and random oracle encodings,
 whose output distribution is indistinguishable from uniformly random.
 Some protocols require a random oracle for security, while others can
 be securely instantiated with a nonuniform encoding.
-When the required encoding is not clear, applications SHOULD use a random oracle.
```

Comment:
Does the document needs to warn implementers something along the lines
"do not be tempted to chose encode_to_curve over hash_to_curve due its performance" ?

Reply:
Sorry, this was a bad edit on my part. I put the sentence back.